style.css

```diff
@@ -84,9 +88,20 @@ form {
     justify-content: space-between;
     align-items: center;
     width: 100%;
+}
+
+li.row {
```

Comment:
@ChrysaLynn 
Since I wanted the `li.row` to have a border-bottom, but not the `a.row`, I split this up here. Now there's a selector for `.row` and one for `li.row` so that we can re-use the `row` class without just adding border upon border.